contributing_to_docs/term_glossary.adoc

```diff
@@ -337,13 +337,10 @@ allowed for use across all distribution documentation:
 
 Usage: OpenShift CLI (`oc`)
 
-The `oc` tool is the command line interface of OpenShift 3 and 4.
+The `oc` tool is the command-line interface of OpenShift 3 and 4.
```

Comment:
Out of curiosity, are you changing "command line interface" to "command-line interface" because of the IBM Style Guide? I'm seeing examples of both in a quick Google search. 

Otherwise, LGTM!

Reply:
Yes, ISG says it should be command-line interface.